src/data.js

```diff
@@ -13233,6 +13233,34 @@ module.exports = [
       'Flutter',
     ],
   },
+  {
+    name: 'Simon Lee',
+    description: 'Coding and learning in an infinite loop.',
+    url: 'https://simonhlee97.github.io/uses/',
+    twitter: '@simonhlee97',
+    emoji: '🌽',
+    country: '🇺🇸',
+    computer: 'apple',
+    tags: [
+      'Front End',
+      'Back End',
+      'FreeCodeCamp Seoul',
+      'JavaScript',
+      'WordPress',
+      'React',
+      'VueJS',
+      'SCSS',
+      'TypeScript',
+      'JamStack',
+      'Node.js',
+      'MongoDB',
+      'Postgres',
+      'UI Developer',
+      'web development',
+      'golf handicap: 15',
```

Comment:
Hey, @simonhlee97 will you remove some of these unique tags and add them to your bio if you'd like them to show?

``` JavaScript
  const tags = Object.entries(counts)
    .sort(([, countA], [, countB]) => countB - countA)
    // Only show the tag if this topic has 3 or more people in it
    .filter(([, count]) => count >= 3)
    .map(([name, count]) => ({ name, count }));
```

Right now only tags with 3+ people show.

Reply:
removed those tags. Thank you.